services/movie.py

```diff
@@ -6,6 +7,7 @@
 def get_movies(
     genres_ids: list[int] = None,
     actors_ids: list[int] = None,
```

Comment:
The type hints for `genres_ids` and `actors_ids` should use `Optional[list[int]]` instead of `list[int]` to indicate that these parameters can be `None`.